h5c/vic/src/vic-webapp/src/app/summary-view/summary-view.component.html

```diff
@@ -7,18 +7,29 @@
           <clr-alert-item *ngIf="error.type === 'ssl_cert'">
             <span class="alert-text">
               <!-- TODO: i18n -->
-              Failed to verify the VIC machine server at endpoint https://{{ error.payload }}:8443. Your browser may be blocking the connection because of the untrusted SSL certificate.
+              Failed to verify the VIC machine server at endpoint https://{{ error.payload }}:8443. Please check the following:<br/>
```

Comment:
This is error prone having the same text here. We should put it into a single include for better maintainability